CHANGELOG.md

```diff
@@ -1,3 +1,68 @@
+# v1.29.0 Release - 07/29/2021
+Note: This release comes with a new config version, `v2beta20`. To upgrade your skaffold.yaml, use `skaffold fix`. If you choose not to upgrade, skaffold will auto-upgrade as best as it can.
+
+Highlights:
+* Skaffold master -> main [#6263](https://github.com/GoogleContainerTools/skaffold/pull/6263)
+* Add state to event handler to track `logrus` output events [#6272](https://github.com/GoogleContainerTools/skaffold/pull/6272)
+* Remove skaffoldLogs endpoint, consolidate into events endpoint [#6296](https://github.com/GoogleContainerTools/skaffold/pull/6296)
```

Comment:
Event API v2 has not being communicated to users. 
Can we move this to updates section

Reply:
Done

Reply:
I don't think this was done